README.md

```diff
@@ -164,6 +166,95 @@ geographic region in support of yourproject’s security and compliance needs.
 See [Running in particular zone/region doc](docs/running_in_particular_zone_region.md).
 
 
+## Running VCF files to BigQuery Preprocessor
+
+Similarly, the preprocessor can be ran using docker or directly from the source.
+
+### Using docker
+
+First, set up the preprocessor pipeline configurations shown below and save it
+as `vcf_to_bigquery_preprocess.yaml`. The parameters that you need to replace
+are:
+
+* `my_project`: This is your project name that contains the BigQuery dataset.
+* `gs://my_bucket/vcffiles/*.vcf`: A location in Google Cloud Storage where the
+  VCF file are stored. You may specify a single file or provide a pattern to
+  load multiple files at once.
+* `report_path`: This can be any path in Google Cloud Storage that your project
+  has write access to. It is used to store the preprocess report.
+* `resolved_headers_path`: Optional. This can be any path in Google Cloud
+  Storage that your project has write access to. It is used to store the
+  resolved headers which can be further used as a representative header for VCF
```

Comment:
nit: I think this is more readable "...used as a representative header (via `--representative_header_file`) for the `[VCF to BigQuery pipeline](#loading-vcf-files-to-bigquery)`."